.github/workflows/nightly-rosetta-t5x-build-test.yaml

```diff
@@ -31,79 +31,120 @@ permissions:
 jobs:
```

Comment:
Same comment about inputs.BASE_IMAGE's default/description in rosetta pax's build-test workflow